fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,271 @@
+                      subplots: bool = True,
+                      plot_error_labels: bool = False,
+                      text_precision_digits: int = 4,
+                      text_fontsize: int = 8,
+                      text_color: str = "black",
+                      text_ha: str = "center",
+                      **kwargs
+                      ):
+    """Visualization for metrics with statistical error bounds.
+
+    Plots a given metric and its given error (as described by the `error_bars` or `conf_intervals`)
+
+    This function takes in a :class:`fairlearn.metrics.MetricFrame` with precomputed metrics
+    and metric errors and a `error_bars` or `conf_intervals` array to interpret the columns
+    of the :class:`fairlearn.metrics.MetricFrame`.
+
+    The items at each index of the given `metrics` array and given `error_bars` or `conf_intervals`
+    array should correspond to a pair of the same metric and metric error, respectively.
+
+    Parameters
+    ----------
+    metric_frame : fairlearn.metrics.MetricFrame
+        The collection of disaggregated metric values, along with the metric errors.
+
+    kind : str, optional
+        The type of plot to display. i.e. "bar", "line", etc.
+        List of options is detailed in :meth:`pandas.DataFrame.plot`
+
+    metrics : str or list of str
+        The name of the metrics to plot.
+        Should match columns from the given :class:`fairlearn.metrics.MetricFrame`.
+
+    error_bars : str or list of str
+        The name of the error metrics to plot.
+        Should match columns from the given :class:`fairlearn.metrics.MetricFrame`.
+        Error bars quantify the bounds relative to the base metric.
+
+        Example:
+            If the error bar for a certain column is :code:`[0.01, 0.02]`
+            and the metric is 0.6,
+            then the plotted bounds will be :code:`[0.59, 0.62]`
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper errors. i.e. :code:`[0.01, 0.02]`
+
+    conf_intervals : str or list of str
+        The name of the confidence intervals to plot.
+        Should match columns from the given :class:`fairlearn.metrics.MetricFrame`.
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper bounds. i.e. :code:`[0.59, 0.62]`
+
+    subplots : bool, optional
+        Whether or not to plot metrics on separate subplots
+
+    plot_error_labels : bool, optional
+        Whether or not to plot numerical labels for the error bounds
+
+    text_precision_digits : int, optional
+        The number of digits of precision to show for error labels
+
+    text_font_size : int, optional
+        The font size to use for error labels
+
+    text_color : str, optional
+        The font color to use for error labels
+
+    text_ha : str, optional
+        The horizontal alignment modifier to use for error labels
```

Comment:
default values should be mentioned in API reference (applies to other args, too)